libcodechecker/cmd/cmd_line_client.py

```diff
@@ -5,39 +5,19 @@
 # -------------------------------------------------------------------------
 
 from datetime import datetime
-import getpass
 import json
 import sys
 
-from thrift.Thrift import TApplicationException
-
 import codeCheckerDBAccess
 import shared
-from Authentication import ttypes as AuthTypes
 
-from libcodechecker import session_manager
 from libcodechecker import suppress_file_handler
+from libcodechecker.api_client.client import setup_client
```

Comment:
Minor thing, but I don't like the package name for this. We should make it somehow inline with the not-subcommand-related libcodechecker subfolders, like `libauth` or `libhandlers`.

Maybe it could go straight into `libauth`? Or `libclient`? Something with `lib` on the beginning of it.